x-pack/test/functional/apps/infra/home_page.ts

```diff
@@ -238,12 +238,12 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         await retry.try(async () => {
           const nodesWithValue = await pageObjects.infraHome.getNodesWithValues();
           expect(nodesWithValue).to.eql([
-            { name: 'demo-stack-apache-01', value: 1.4, color: '#6092c0' },
-            { name: 'demo-stack-mysql-01', value: 1.2, color: '#82a7cd' },
-            { name: 'demo-stack-nginx-01', value: 1.1, color: '#93b1d3' },
-            { name: 'demo-stack-redis-01', value: 1, color: '#a2bcd9' },
+            { name: 'demo-stack-apache-01', value: 1.2, color: '#6092c0' },
+            { name: 'demo-stack-mysql-01', value: 1, color: '#93b1d3' },
+            { name: 'demo-stack-nginx-01', value: 0.9, color: '#b2c7df' },
+            { name: 'demo-stack-redis-01', value: 0.8, color: '#b2c7df' },
             { name: 'demo-stack-haproxy-01', value: 0.8, color: '#c2d2e6' },
-            { name: 'demo-stack-client-01', value: 0.6, color: '#f0f4f9' },
+            { name: 'demo-stack-client-01', value: 0.5, color: '#f0f4f9' },
```

Comment:
We lose precision when we ask Elasticsearch to calculate the metrics instead of computing the values in the time series buckets.

Reply:
It depends on the data, but it is expected to be different.

table:
<img width="902" alt="image" src="https://github.com/elastic/kibana/assets/2767137/91cc371f-0bc9-4c32-9f0e-10de8ae178b6">

waffle:
<img width="635" alt="image" src="https://github.com/elastic/kibana/assets/2767137/9a92f36f-6a4a-4d53-bdd1-53410e9096f0">